python/requirements.txt

```diff
@@ -74,4 +74,4 @@ werkzeug
 xlrd
 starlette
 smart_open[s3]
-tqdm
\ No newline at end of file
+tqdm
```

Comment:
Remove this change?

Reply:
Hmm this is weird. I think tqdm should be here originally (because we need it for shuffle.py).. let me see if I am missing sth here

Reply:
Looks like it's just a whitespace change.

Reply:
Ah, that makes sense 😂